shared/cfg/base.cfg

```diff
@@ -592,6 +592,7 @@ netdev_peer_re = "\s{2,}(.*?): .*?\\\s(.*?):"
 #kvm_ver_cmd = "modinfo kvm | grep vermagic | awk '{print $2}'"
 #kvm_userspace_ver_cmd = "grep -q el5 /proc/version && rpm -q kvm || rpm -q qemu-kvm"
 
+qemu_img_command = 'qemu-img create -f qcow2 -o backing_file=%s %s'
```

Comment:
I can understand you are trying to save time and space here.
But there two problems here:
1) When we talk about vm clone, it kind of produces a new copy of the previous vm, so it should be two separated copies. From semantic  aspect,  if using backing file then they are `linked`,it is not a `clone`.
2) From data integrity, If you clone a vm1 using backing file get a vm2, you can not guarantee vm1 does not change alone the test.

Reply:
@hereischen Thanks for comments.

Reply:
As far as I know, there is no mechanism to protect VM1 from being modified in the later process.

Reply:
As per document it is mentioned that the VM1 will be read-only. Even I tried to bring guest with VM1 it failed saying you cannot use the VM1.  We can start using it in avocado in case if VM1 is corrupted or overwritten then it will be a bug.

Reply:
It's not true for `vt_type = qemu`, there is no such a prevention given by qemu, the image corruption is the expectation of that case. And also, I see that the image format has been hardcoded to `qcow2`, it would be another issue that it does not respect the option set by user (the image format variants defined in `shared/cfg/guest-hw.cfg`).

Overall, as @hereischen said, backing does not have the same meaning as cloning, so we cannot reuse the method `clone_image()` to do that.